### PR TITLE
New proto fix

### PR DESCRIPTION
**Pull request by haybankz**
I have 

- [ ] made changes according to the new proto file
- [ ] added permission for photo access for ios

### CODE REVIEW

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -59,5 +59,14 @@
...
         <string>pylons-wallet</string>
     </array>
+
+    <key>UIBackgroundModes</key>
+    <array>
+       <string>fetch</string>
+       <string>remote-notification</string>
+    </array>
+    <key>NSPhotoLibraryUsageDescription</key>
+    <string>Explain why your app uses photo library</string>
```
Comment (kjawadDeveloper1) [inline ios/Runner/Info.plist]: This will cause rejection by the apple

File: lib/easel_provider.dart [hunk 2]
```diff
@@ -279,4 +270,12 @@ class EaselProvider extends ChangeNotifier {
     royaltyController.dispose();
     super.dispose();
   }
+
```
Comment (kjawadDeveloper1) [inline lib/easel_provider.dart]: Remove dead code.

**Review by kjawadDeveloper1 — APPROVED** [review]
Looks good from my side.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Pull request merged**